### PR TITLE
Update exhaustive_feature_selector.py

### PR DESCRIPTION
**Pull request by Haider010**
AttributeError: `np.NINF` was removed in the NumPy 2.0 release. Use `-np.inf` instead. Fixed this Error.

### Pull Request Checklist

- [ ] Added a note about the modification or contribution to the `./docs/sources/CHANGELOG.md` file (if applicable)
- [ ] Added appropriate unit test functions in the `./mlxtend/*/tests` directories (if applicable)
- [ ] Modify documentation in the corresponding Jupyter Notebook under `mlxtend/docs/sources/` (if applicable)
- [ ] Ran `PYTHONPATH='.' pytest ./mlxtend -sv` and make sure that all unit tests pass (for small modifications, it might be sufficient to only run the specific test file, e.g., `PYTHONPATH='.' pytest ./mlxtend/classifier/tests/test_stacking_cv_classifier.py -sv`)
- [ ] Checked for style issues by running `flake8 ./mlxtend`

### CODE REVIEW

**Comment by d-kleine** [comment]
Has been fixed with #1107 already, this issue can be closed

**Comment by gedeck** [comment]
@d-kleine  This fixes another occurrence of np.NINF in `ExhaustiveFeatureSelector`

**Comment by d-kleine** [comment]
@gedeck Thanks for pointing that out - yeah, you are right. I overlooked that there are two different feature selector implementations (exhaustive vs. sequential). About how this issue can be fixed, I would like to leave the final decision to @rasbt.

**Comment by rasbt** [comment]
Thanks for the PR, and I might have missed that previously, sorry!

The fix looks good, but I think it might break in earlier NumPy versions then (not everyone has transitioned to 2.0 yet I assume). The fix in #1107 by @d-kleine  was actually ideal. If we could apply it here too, that'd be great.

**Comment by d-kleine** [comment]
@Haider010 or @rasbt As I cannot contribute to this PR, please feel free to take my changes from #1107 for fixing this issue 🙂

File: mlxtend/feature_selection/exhaustive_feature_selector.py [hunk 1]
```diff
@@ -483,7 +483,7 @@ def ncr(n, r):
         return self
 
     def finalize_fit(self):
-        max_score = np.NINF
+        max_score = -np.inf
```
Comment (gedeck) [inline mlxtend/feature_selection/exhaustive_feature_selector.py:486]:
```suggestion
        if np.__version__ < "2.0":
            ninf = np.NINF
        else:
            ninf = -np.inf
        max_score = ninf
```

Comment (rasbt) [reply mlxtend/feature_selection/exhaustive_feature_selector.py:486]: Thanks, this should be it :)

**Review by rasbt — APPROVED** [review]

**Comment by d-kleine** [comment]
Could you please release a new package version for `pip` users?

**Pull request merged**